classes/event/start_document_conversion.php

```diff
@@ -46,6 +46,6 @@ public static function get_name() {
     }
 
     public function get_description() {
-        return get_string('event:start_document_conversion', 'fileconverter_librelambda');
+        return "The conversion with id '{$this->other['id']}' has been commenced with status '{$this->other['status']}'.";
```

Comment:
Do we really need status here? Will it be different?

Reply:
In theory it can be error if something went wrong: https://github.com/catalyst/moodle-fileconverter_librelambda/blob/d90802effe3649cff36b7a568f0809df7f633f54/classes/converter.php#L346